zeroconf.c

```diff
@@ -385,9 +385,12 @@ OpcUa_StatusCode OPCUA_DLLCALL ualds_zeroconf_registerInternal(OpcUa_Void*  pvCa
         /* replace [gethostname] in own server name */
         if (bOwnRegistration != OpcUa_False)
         {
-            char hostname[UALDS_CONF_MAX_URI_LENGTH] = {0};
+            // additional 6 chars for '.local'
+            char hostname[UALDS_CONF_MAX_URI_LENGTH+6] = {0};
             ualds_platform_getfqhostname(hostname, sizeof(hostname));
```

Comment:
this function is supposed to give the fully qualified host name.
adding a .local to the fqdn string would be wrong.